Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/azuredeploy.json

```diff
@@ -0,0 +1,331 @@
+        {
+            "type": "Microsoft.Storage/storageAccounts/blobServices",
+            "apiVersion": "2019-06-01",
+            "name": "[concat(variables('ProducerFunctionName'), '/default')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Storage/storageAccounts', tolower(variables('ProducerFunctionName')))]"
+            ],
+            "sku": {
+                "name": "Standard_LRS",
+                "tier": "Standard"
+            },
+            "properties": {
+                "cors": {
+                    "corsRules": []
+                },
+                "deleteRetentionPolicy": {
+                    "enabled": false
+                }
+            }
+        },
+        {
+            "type": "Microsoft.Storage/storageAccounts/queueServices",
+            "apiVersion": "2022-05-01",
+            "name": "[concat(variables('ProducerFunctionName'), '/default')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Storage/storageAccounts', tolower(variables('ProducerFunctionName')))]"
+            ],
+            "properties": {
+                "metadata": {}
+            }
+        },
+        {
+            "type": "Microsoft.Web/sites",
+            "apiVersion": "2018-11-01",
+            "name": "[variables('ProducerFunctionName')]",
+            "location": "[resourceGroup().location]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Web/serverfarms', variables('hostingPlanName'))]",
+                "[resourceId('Microsoft.Storage/storageAccounts', tolower(variables('ProducerFunctionName')))]"
+            ],
+            "kind": "functionapp",
+            "identity": {
+                "type": "SystemAssigned"
+            },
+            "properties": {
+                "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', variables('hostingPlanName'))]",
+                "name": "[variables('ProducerFunctionName')]",
+                "httpsOnly": true,
+                "clientAffinityEnabled": true,
+                "alwaysOn": true,
+                "reserved": true
+            },
+            "resources": [{
+                "type": "config",
+                "apiVersion": "2018-11-01",
+                "name": "appsettings",
+                "dependsOn": [
+                    "[concat('Microsoft.Web/sites/', variables('ProducerFunctionName'))]"
+                ],
+                "properties": {
+                    "CohesityQueueName": "[variables('QueueName')]",
+                    "keyVaultName": "[variables('ProducerFunctionName')]",
+                    "Workspace": "[parameters('Workspace')]",
+                    "AzureWebJobsStorage": " [concat('DefaultEndpointsProtocol=https;AccountName=', toLower(variables('ProducerFunctionName')),';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', toLower(variables('ProducerFunctionName'))), '2019-06-01').keys[0].value, ';EndpointSuffix=',toLower(variables('StorageSuffix')))]",
```

Comment:
You have the same parameter in the KeyVault section (see ln 125). Please leave one of them - either this or read it from Keyvault.

Reply:
which one? AzureWebJobsStorage?

Reply:
This comment is outdated.